<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>779</width>
    <height>660</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>450</y>
      <width>781</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-3</y>
      <width>781</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/images/20220809104550.png);</string>
    </property>
    <widget class="QWidget" name="layoutWidget_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>-1</y>
       <width>461</width>
       <height>71</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>IP地址：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtIp">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Normal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>端口：</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="txtPort">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushConnect">
        <property name="text">
         <string>连接</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="layoutWidget_3">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>-1</y>
       <width>321</width>
       <height>71</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>间隔时间(s):</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
          <horstretch>50</horstretch>
          <verstretch>20</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="timeBut">
        <property name="text">
         <string>启动定时</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QFrame" name="Frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>390</y>
      <width>781</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/images/20220808141751.jpg);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QFontComboBox" name="fontCbx">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>250</width>
         <height>30</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>11</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="sizeCbx">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>50</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>50</width>
         <height>30</height>
        </size>
       </property>
       <property name="currentIndex">
        <number>7</number>
       </property>
       <item>
        <property name="text">
         <string>15</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>9</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>10</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>11</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>12</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>13</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>14</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>15</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>16</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>17</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>18</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>19</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>20</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>21</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>22</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>23</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>24</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>25</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="italicTBtn">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>35</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>35</height>
        </size>
       </property>
       <property name="toolTip">
        <string>斜体</string>
       </property>
       <property name="styleSheet">
        <string notr="true">background-image: url(:/images/xieti.jpg);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="boldTBtn">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>35</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>35</height>
        </size>
       </property>
       <property name="toolTip">
        <string>加粗</string>
       </property>
       <property name="styleSheet">
        <string notr="true">background-image: url(:/images/cuti.jpg);</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="underlineTBtn">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>35</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>35</height>
        </size>
       </property>
       <property name="toolTip">
        <string>下划线</string>
       </property>
       <property name="styleSheet">
        <string notr="true">background-image: url(:/images/xiahuaxian.jpg);</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="colorTBtn">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>35</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>35</height>
        </size>
       </property>
       <property name="toolTip">
        <string>字体颜色</string>
       </property>
       <property name="styleSheet">
        <string notr="true">background-image: url(:/images/yanse.jpg);</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>35</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>35</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">image: url(:/images/jietu.jpg);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="saveTBtn">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>35</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>35</height>
        </size>
       </property>
       <property name="toolTip">
        <string>保存</string>
       </property>
       <property name="styleSheet">
        <string notr="true">background-image: url(:/images/save.jpg);</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="clearTBtn">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>35</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>35</height>
        </size>
       </property>
       <property name="toolTip">
        <string>删除</string>
       </property>
       <property name="styleSheet">
        <string notr="true">background-image: url(:/images/delete.jpg);</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTextEdit" name="txtData">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>781</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushSent">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>630</y>
      <width>101</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 170, 0);</string>
    </property>
    <property name="text">
     <string>发送数据</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>560</y>
      <width>72</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
